mender-update/http_resumer/http_resumer.cpp

```diff
@@ -420,7 +427,8 @@ error::Error DownloadResumerClient::ScheduleNextResumeRequest() {
 				auto err_user = http::MakeError(
 					http::DownloadResumerError,
 					"Failed to schedule the next resumer call: " + next_call_err.String());
-				DisableWithError(err_user);
+				logger_.Error(err_user.String());
+				Cancel();
```

Comment:
This cancels the whole request, doesn't it? Shouldn't it rather wait and retry here?

Reply:
This is while re-scheduling and an error on `http::Client.AsyncCall` -> I consider it unrecoverable. In which cases would a re-schedule help?

Reply:
You're probably right.. Probably..

But could it happen for example if all interfaces are down? I have no idea if Boost would report that immediately or through the handler. Maybe it makes sense to be fault tolerant also here?

Reply:
Okey, we'll retry also then.